src/popup/popup.js

```diff
@@ -16,12 +16,10 @@ new Flickity(document.getElementById('flickity'), {
 	on: { change: (slideNumber) => {
 		if (slideNumber === 5) {
 			document.getElementById('intro-exit').innerHTML = 'done';
-			document.getElementById('intro-exit').style.color = '#EF5E92';
-			document.getElementById('intro-exit').style.opacity = 1;
+            document.getElementById('intro-exit').style.color = '#C31354';
 		} else {
 			document.getElementById('intro-exit').innerHTML = 'skip';
 			document.getElementById('intro-exit').style.color = '#000000';
-			document.getElementById('intro-exit').style.opacity = 0.5;
```

Comment:
The opacity for "skip" is set to 0.5 in the css - this just sets it back to 0.5 if we navigate to the last tutorial page, where "done" is displayed instead and then head back to another tutorial page. Removing it means that the opacity starts at 0.5 and ends up being 1 after we navigate to the "done" page. We should either keep it at 0.5 (add this line back in) or change the css to use opacity = 1